tasks/HelpMeCarry.tex

```diff
@@ -0,0 +1,76 @@
+\section{Help Me Carry [Party Host]}
+The robot helps a guest to carry some luggage to his/her car.
+
+\subsection{Main Goal}
+Robot helps a guest to carry a bag to the car which is parked outside and comes back home.
+
+\subsection{Focus}
+It includes understanding pointing gesture, object identification and manipulation, navigation in known and unknown environments, person-following while carrying a luggage, and social navigation.
+
+\noindent\textbf{Reward:} 500 pts for carrying the bag to the car's location and coming back home.
+
+\subsection{Bonus rewards}
+\begin{enumerate}[nosep]
+	\item Find the bag by understanding the pointing behaviour (100 pts).
+	\item Avoid all the obstacles outdoor (400 pts).
+\end{enumerate}
+
+
+\subsection{Setup}
+\begin{itemize}
+	\item \textbf{Location}: The test takes place inside and outside the arena.
+
+	\item \textbf{Start Location}: The robot starts at a predefined location in the living room.
+
+	\item \textbf{Guest}: There is only one guest in the house, standing in front of the robot at the start.
+	
+	\item \textbf{Bags' Location:} There are at least two bags placed between one to two meters  away from the guest's start position and both should be visible to the robot.
+	
+	\item \textbf{Pointing to the Bag:} When the test starts, the guest points to his/her bag and requests the robot to carry it to the car.
+	
+	\item \textbf{Outdoor Obstacles:} The robot will face multiple obstacles outside. The obstacles may include a person, a group of people, a stationary object or a moving object.
+\end{itemize}
+
+\subsection{Additional rules and remarks}
+\begin{enumerate}[nosep]
+	\item \textbf{Deus ex Machina:} Score reduction for requesting human assistance is applied as follows.
+	\begin{itemize}
+		\item help in finding the bag (-100 pts).
+		\item help in carrying the bag (-100 pts).
+		\item help in tracking the guest (-200pts).
+		\item help in going back home (-100pts).
```

Comment:
How?

Reply:
Let's remove the penalty for not finding the bag autonomously because that's a bonus task. Similarly, help in going back to home should not be penalized because as per Section 3.8.2 no bonus points are scored in case of any help. 

Besides, help in carrying the bag can be addressed using the DeM rules in Section 3.8.2 of the rulebook. So we can also drop it from the DeM list here.

For help in tracking the operator there can be many ways and listing all of them seems infeasible. I can think of, robot can request to wave, request to call its name loudly, stand in front of it, press a button on its screen, hold its hand and drag him along, push him from back in the right direction, etc. So to keep it simple we can have a single rule to assign a fixed penalty for any help in tracking the operator. And since tracking the person is the most important component of this task let's have a heavy penalty for that.

What would you say based on your previous experiences? How did people work around if they got stuck in person following tasks?

Reply:
> Let's remove the penalty for not finding the bag autonomously because that's a bonus task.

Agreed.

> help in going back to home should not be penalized [...] no bonus points are scored in case of any help.

A partial bonus can be awarded if the robot returns home with assistance (e.g. lost operator who needed to stop and wave). That's preferable over not returning home but shall score less than returning without losing track of the operator.

> carrying the bag can be addressed using the DeM [...] So we can also drop it [...]

Agreed. Nonetheless it would come handy to have the handover score reduction in the scoresheet.

> listing all of them seems infeasible

Agreed, how it is done is irrelevant IF THE OPERATOR DOES NOT TOUCH THE ROBOT. Waving, calling, and other natural interactions, are ok (20% reduction), while doing gestures (e.g.  \o/ or  Lo⅃ ) might incur in 40% reduction. Taking robot's hand and slightly pull wasn't allowed before, but now it is. This is mechanically guiding the robot, so I would apply at least 80% reduction if not 100% reduction.

As I said above, it would come handy to have the reductions in the scoresheet.

### Example
**Main goal:** Take the bag to the car (300pts)
**Bonus goal:** Come back home (200pts)
**Deus Ex Machina:** (These might be only in the scoresheet)
- Losing track of the operator during the *guiding phase* reduces the score in 50pts
- Non-natural interaction during the *guiding phase* reduces the score in 100pts
- Touching the robot during a *guiding phase* neglects the score
- Requesting indications during the *Come back home* reduces the bonus' score in 50pts

Reply:
> A partial bonus can be awarded if the robot returns home with assistance (e.g. lost operator who needed to stop and wave). That's preferable over not returning home but shall score less than returning without losing track of the operator.

Section 3.8.2 says,

Bonuses: No bonus points can be scored when the robot requests help to solve part of a task that normally would grant a bonus.

If we are going to keep this rule then any help in solving the bonus task should result in zero points for that bonus task.


> Agreed, how it is done is irrelevant IF THE OPERATOR DOES NOT TOUCH THE ROBOT. Waving, calling, and other natural interactions, are ok (20% reduction), while doing gestures (e.g. \o/ or Lo⅃ ) might incur in 40% reduction. Taking robot's hand and slightly pull wasn't allowed before, but now it is. This is mechanically guiding the robot, so I would apply at least 80% reduction if not 100% reduction.

> As I said above, it would come handy to have the reductions in the scoresheet.

Agree.

Reply:
> * Losing track of the operator during the _guiding phase_ reduces the score in 50pts

If the robot looses track and then finds the operator without any human help then we should not penalize it.

Reply:
> Section 3.8.2 says...

I wrote Section 3.8.2, but at the beginning of §3 it says:

> Every rule in this section can be considered to implicitly include the term *"unless stated other-
wise"*, meaning that additional or contrary rules in particular test specifications have a higher
priority than those mentioned herein in the general rules and regulations.

From my previous experience, if a bonus reward is too challenging teams won't attempt it unless there is partial score involved.

Being said that, it is your test, you decide whether to allow partial scoring on bonuses.